Tools/TestWebKitAPI/Tests/WGSL/ConstLiteralTests.cpp

```diff
@@ -43,38 +42,36 @@
     NumberType expectedValue;
 };
 
-@interface ConstLiteralTests : XCTestCase
+namespace TestWebKitAPI {
```

Comment:
Does TestWTFAPI use the `TestWebKitAPI` namespace? Should we make a TestWGSLAPI namespace?